### PR TITLE
Correct COMPRESS_CLEAN_CSS_* name in the docs

### PR DESCRIPTION
**Pull request by kyrias**
Signed-off-by: Johannes Löthberg <johannes@kyriasis.com>

### CODE REVIEW

File: compressor/conf.py [hunk 1]
```diff
@@ -45,8 +45,8 @@ class CompressorConf(AppConf):
     YUGLIFY_BINARY = 'yuglify'
     YUGLIFY_CSS_ARGUMENTS = '--terminal'
     YUGLIFY_JS_ARGUMENTS = '--terminal'
-    CLEAN_CSS_BINARY = 'cleancss'
-    CLEAN_CSS_ARGUMENTS = ''
+    COMPRESS_CLEAN_CSS_BINARY = 'cleancss'
+    COMPRESS_CLEAN_CSS_ARGUMENTS = ''
```
Comment (diox) [inline compressor/conf.py]: appconf transforms all properties set in that class, adding the `COMPRESS_` prefix before turning them in settings, so this is wrong.

File: docs/settings.txt [hunk 2]
```diff
@@ -146,11 +146,11 @@ Backend settings
 
       A filter that passes the CSS content to the `clean-css`_ tool.
 
-      .. attribute:: CLEAN_CSS_BINARY
+      .. attribute:: COMPRESS_CLEAN_CSS_BINARY
```
Comment (diox) [inline docs/settings.txt]: This one however is right, good call.

File: compressor/conf.py (hunk 1 above)
Comment (kyrias) [inline compressor/conf.py]: Ah, sorry 'bout that.

**Comment by kyrias** [comment]
Amended and pushed

**Comment by diox** [comment]
Can you amend your commit message and pull request title, since the code was correct, only the docs were wrong ? Thanks.

**Comment by kyrias** [comment]
Gah, forgot that. Fixed now.

**Comment by diox** [comment]
Thanks!

**Pull request merged**